### PR TITLE
Simplify Get started section

### LINKED ISSUE #716: Getting started docs: Make it clear to use the releases rather than the master branch
Currently, the README.md seems to indicate to use the master branch rather than the releases.
We want people to use the releases, and this should be very clear.

### PR DESCRIPTION
**Pull request by MiroslavRepka**
Closes #716

Some work done toward #720

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,21 +75,23 @@ For adding support for other cloud providers, open an issue or propose a PR.
...
 
-   ```sh
-   kubectl apply -k manifests/claudie
-   ```
+    ```sh
+    wget https://github.com/berops/claudie/releases/latest/download/claudie.zip && unzip claudie.zip -d claudie
+    ```
 
-2. provide your own manifest via a Kubernetes Secret.
+2. Deploy Claudie Kubernetes into a Kubernetes cluster.
```
Comment (MarioUhrik) [inline README.md:84]: `Deploy Claudie Kubernetes into a Kubernetes cluster.`
probably just
`Deploy Claudie into a Kubernetes cluster.`

**Review by MarioUhrik — CHANGES_REQUESTED** [review]
I like this, it's a big improvement.
One tiny comment though, up for your consideration

**Review by MarioUhrik — APPROVED** [review]
LGTM at the moment, but I'm looking forward to see what Bernard can come up with to further improve it

**Comment by MiroslavRepka** [comment]
>I am going to run through the getting started section as well, I'd be happy if it was that good that this was the only improvement we could've found.

Should I remove #720 for the time being? @bernardhalas

**Comment by bernardhalas** [comment]
Yes, please. Just a reference between the issue and the PR is perfect, let's not close #720 yet.

**Pull request merged**